### PR TITLE
#52 Toggle feeds

### PR DESCRIPTION
**Pull request by ak9999**
Not ready for merging yet. This is related to #52.

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -54,7 +57,7 @@
                 while (list.childNodes.length > 0) {
                     list.removeChild(list.firstChild);
                 }
-                XHR('/feed/events/cool?page=' + page, 'GET', undefined,
+                XHR(`/feed/events/${event_type}?page=` + page, 'GET', undefined, // Using template literals
```
Comment (akellehe) [inline templates/index.html]: Not supported on either IE or Android platforms

Comment (pisoiasul) [inline templates/index.html]: Ha I didn't know about this. ES6 ?

Comment (akellehe) [inline templates/index.html]: I know! I didn't either. It's pretty cool. 👍 to @ak9999 for finding a pretty neat feature.

Comment (ak9999) [inline templates/index.html]: Is ES6 the same as ECMAScript 2015? Oh well.

**Comment by akellehe** [comment]
@ak9999 how are things going here?

**Comment by ak9999** [comment]
Frozen because I'm still learning JS.

**Pull request closed without merging**